fh-android-sdk/src/main/java/org/json/fh/HTTP.java

```diff
@@ -147,19 +147,17 @@ public static JSONObject toJSONObject(String string) throws JSONException {
      *    "Reason-Phrase": "OK" (for example)
      * }
      * </pre>
-     * 
+     *
      * Any other members of the JSONObject will be output as HTTP fields.
      * The result will end with two CRLF pairs.
-     * 
+     *
      * @param o A JSONObject
      * @return An HTTP header string.
-     * @throws JSONException this will be thrown if there is an error parsing the JSON if the object does not contain enough
-     *             information.
+     * @throws JSONException if there is an error parsing the JSON if the object does not contain
```

Comment:
Should the second `if` be replaced with `or`? EG `...or the object does not contain...`